.cirrus.yml

```diff
@@ -0,0 +1,31 @@
+BUILD_TEST_TASK_TEMPLATE: &BUILD_TEST_TASK_TEMPLATE
+  test_pact_mock_server_cli_script: |
+    cd rust/pact_mock_server_cli
+    cargo build --target ${BUILD_TARGET} --release
+    ../target/${BUILD_TARGET}/release/pact_mock_server_cli --help
+    rm -rf ../target/
+  test_pact_verifier_cli_script: |
+    cd rust/pact_verifier_cli
+    cargo build --target ${BUILD_TARGET} --release
+    ../target/${BUILD_TARGET}/release/pact_verifier_cli --help
+
+
+linux_arm64_task: 
+  arm_container:
+    image: rustlang/rust:nightly
```

Comment:
Is there a reason to need nightly here, I would prefer to run on stable and/or beta

Reply:
Yeah I would have thought that too, I chose it as it was from the release flow where these get built

https://github.com/pact-foundation/pact-reference/blob/1116eff9850781c802217bc67481a84474d3dc5c/.github/workflows/release.yml#L49

Although that only appears before the FFI run (which is maybe more suspect) especially in a release run

kicked off a run against stable tag now - https://cirrus-ci.com/build/4982626734309376